src/v2/Components/ArtworkFilter/Utils/isDefaultFilter.tsx

```diff
@@ -20,6 +21,8 @@ export const isDefaultFilter: (
       name === "artistNationalities" ||
       name === "materialsTerms":
       return value.length === 0
+    case !!defaultValues[name]:
```

Comment:
This line is kinda hard to understand at a glance; can we assign it to a self-describing variable name and then use that here instead?